salesforce_api/utils/bulk.py

```diff
@@ -38,7 +37,6 @@ def get_csv_file(self) -> io.StringIO:
         writer.writerows([
             entry.values()
             for entry in self.entries
-            if ''.join([str(x) for x in entry.values() if x is not None]) != ''

```

Comment:
It will already fail above `if not self._check_headers()`, which does this same check, so removed this.